content/2017.04.getting-started-with-elm.rst

```diff
@@ -0,0 +1,487 @@
+Getting started with Elm
+########################
+
+:lang: en
+:date: 2017-04-13
+:url: getting-started-with-elm
+:summary: What is Elm? how does it work? How can I try it really fast?
+
+
+Introduction
+============
+
+The past two weeks we have been traveling and meeting a bunch of
+developers.
+
+It was a chance for us to tell them about Elm through talks at
+conferences and to present a short one-hour workshop.
+
+You can find the slides of the two talks here:
+
+- `a short introduction to Elm by Niko <http://slides.com/n1k0/elm>`_
+- `Talking JSON with Elm by Natim <http://natim.github.io/django-slides/elm-json/>`_
+
+However, this article is a summary of the take aways of this workshop
+for those who would like to know a bit more about Elm.
+
+
+What is Elm and why should I be interested?
+===========================================
+
+Elm is a toolkit that is focused on making sure your app state and the
```

Comment:
Hey Areski 🍻

I usually describe Elm as a development platform, providing a programming language, a compiler, an architecture and tooling around. It's longer but quite descriptive.